arex-web-api/src/main/java/com/arextest/web/api/service/controller/ReportQueryController.java

```diff
@@ -375,4 +375,10 @@ public Response syncResponse(@Valid @RequestBody OverwriteContractRequestType re
         return ResponseUtils.successResponse(schemaInferService.overwriteContract(requestType));
     }
 
+    @GetMapping("/listCategoryType")
+    @ResponseBody
+    public Response listCategoryType() {
```

Comment:
Receive the configuration related interface under the config directory
Can hide the main mock category type